### ISSUE TITLE
[FEAT] Create Hook - useKeyPress()

### ISSUE THREAD

**Issue by DavidHDev** [DavidHDev/react-haiku#62]
**Description**
A hook that detects when a specific key or combination of keys is pressed.

**Acceptance Criteria**
- [ ] Clear Documentation: relevant examples added to showcase the capability of the hook
- [ ] Respects Standards: the documentation should go hand-in-hand with the style and implementation of other docs pages

**Comment by A-4-Atom** [comment]
Hi, the useKeyPress() hook is already created but it has no documentation, I'd like to work on the documentation of this hook.
@DavidHDev

**Comment by DavidHDev** [comment]
Go for it! i'll change the title of this issue 👍

**Comment by DavidHDev** [comment]
@A-4-Atom done, also updated the acceptance criteria for this

**Comment by A-4-Atom** [comment]
I was trying to create a demo component but it looks like that **useKeyPress()** hook doesn't exist on **react-haiku** that's published on npm.
Code for the demo component:
```
import { useKeyPress } from 'react-haiku';
import React, { useState } from 'react';
import './demo.css';

export const UseKeyPressDemo = () => {
  const [didKeyPress, setDidKeyPress] = useState(false);
   useKeyPress(['Control', 'Shift', 'A'], (e) => {
     setDidKeyPress(true);
   })

  return (
    <div className="demo-container-start">
        {didKeyPress && <p>{`You pressed - Control + Shift + A`}</p>}
      <p>Press Control + Shift + A</p>
    </div>
  );
};
```

I noticed that while the hook is exported from the file _./lib/index.ts_  , but the same thing does not exist on the npm  package that's published, take a look at the _./dist/index.js_ [here](https://www.npmjs.com/package/react-haiku?activeTab=code).

@DavidHDev

**Comment by DavidHDev** [comment]
@A-4-Atom yes, it's probably not released in a version, but you can link a local version of the library to your node_modules or just use the local file for testing purposes.
